packages/api-explorer/src/components/ExploreType/ExploreProperty.tsx

```diff
@@ -0,0 +1,262 @@
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React, { FC } from 'react'
+import {
+  FlexItem,
+  IconNames,
+  Icon,
+  Tree,
+  TreeItem,
+  Tooltip,
+  Space,
+} from '@looker/components'
+import { IProperty } from '@looker/sdk-codegen'
+import ReactMarkdown from 'react-markdown'
+import {
+  expandable,
+  ExploreTypeLink,
+  pickType,
+  pickTypeProps,
+  typeIcon,
+} from '.'
+
+interface TipIconProps {
+  show: boolean
+  tip: string
+  icon: IconNames
+  title: string
+}
+
+/**
+ * Show a tip with icon if show is true
+ * @param show true to show tip and icon
+ * @param tip to display on hover
+ * @param icon name to use
+ * @param title for icon
+ * @constructor
+ */
+export const TipIcon: FC<TipIconProps> = ({ show, tip, icon, title }) => {
+  if (!show) return <></>
+  return (
+    <Tooltip content={tip}>
+      <Icon name={icon} size="xsmall" content={tip} title={title} />
+    </Tooltip>
+  )
+}
+
+/**
+ * Interface shared by several ExploreProperty components
+ */
+interface ExplorePropertyProps {
+  /** property to explore */
+  property: IProperty
+  /** the current level of the hierarchy */
+  level?: number
+  /** the maximum depth to expanded nested types. -1 = all (default), 0 = no expansion */
+  maxDepth?: number
+  /** open all nodes immediately? */
+  openAll?: boolean
+}
+
+/**
+ * Show required status if a property is required
+ * @param property to describe
+ * @constructor
+ */
+export const ExplorePropertyRequired: FC<ExplorePropertyProps> = ({
+  property,
+}) => {
+  const tip = `${property.fullName} is required`
+  return (
+    <TipIcon
+      show={property.required}
+      icon="Check"
+      tip={tip}
+      title="required property"
+    />
+  )
+}
+
+/**
+ * Show deprecated status if a property is deprecated
+ * @param property to describe
+ * @constructor
+ */
+export const ExplorePropertyDeprecated: FC<ExplorePropertyProps> = ({
+  property,
+}) => {
+  const tip = `${property.fullName} is deprecated`
+  return (
+    <TipIcon
+      show={property.deprecated}
+      icon="Minus"
+      tip={tip}
+      title="deprecated property"
+    />
+  )
+}
+
+/**
+ * Show read-only status if a property is read-only
+ * @param property to describe
+ * @constructor
+ */
+export const ExplorePropertyReadOnly: FC<ExplorePropertyProps> = ({
+  property,
+}) => {
+  const tip = `${property.fullName} is read-only`
+  return (
+    <TipIcon
+      show={property.readOnly}
+      icon="LockClosed"
+      tip={tip}
+      title="read-only property"
+    />
+  )
+}
+
+/**
+ * Display the property description if it's assigned
+ * @param property to describe
+ * @constructor
+ */
+export const ExplorePropertyDescription: FC<ExplorePropertyProps> = ({
+  property,
+}) => {
+  if (!property.description) return <></>
+  return <ReactMarkdown source={property.description} />
+}
```

Comment:
Please see below for original comment
```suggestion
export const ExplorePropertyDescription: FC<ExplorePropertyProps> = ({
  property,
}) => (
  <>{property.description && <ReactMarkdown source={property.description} />}</>
)
```